sdk/keyvault/Azure.Security.KeyVault.Keys/src/EcKeyCreateOptions.cs

```diff
@@ -2,18 +2,40 @@
 // Licensed under the MIT License. See License.txt in the project root for
 // license information.
 
-using System;
-using System.Collections.Generic;
-
 namespace Azure.Security.KeyVault.Keys
 {
+    /// <summary>
+    /// Represent the key specific attributes needed in order to create a EC key.
+    /// </summary>
     public class EcKeyCreateOptions : KeyCreateOptions
     {
+        /// <summary>
+        /// Name of the key.
+        /// </summary>
         public string Name { get; set; }
+
+        /// <summary>
+        /// The type of key to create.
+        /// </summary>
         public KeyType KeyType { get; private set; }
+
+        /// <summary>
+        /// Elliptic curve name. For valid values, see <see cref="KeyCurveName"/>. Possible values include: 'P-256', 'P-384',
```

Comment:
Yeah, from what I understand. It depends on the EC crypto algorithm that your key uses.
The only MS information I could find (not sure if it will be useful) is https://docs.microsoft.com/en-us/rest/api/keyvault/createkey/createkey#jsonwebkeycurvename ).

@schaabs is there a more helpful resource for this?

Reply:
hmm maybe this one? https://docs.microsoft.com/en-us/azure/key-vault/about-keys-secrets-and-certificates#ec-algorithms